### DISCUSSION TITLE
Hardware for Alarm

### DISCUSSION THREAD

**Discussion by Marcoletto** [gicamm/homeassistant-comelit#58] [Q&A] [answered]
Dear,
I've got a vedo alarm with only the PCB for "voice" ( for activate it ).

What is the correct PCB that I have to install?

Thank you

**Comment by gicamm** [comment, marked as answer]
Hi,
If you want to manage the alarm with your smartphone, web interface or this component, you need the [VEDOIP](https://pro.comelitgroup.com/it-it/prodotto/vedoip).
